samples/Diagnostics.StatusCodes.Mvc/Startup.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.Diagnostics;
+using Microsoft.AspNet.Routing;
+using Microsoft.Framework.DependencyInjection;
+
+namespace Diagnostics.StatusCodes.Mvc
+{
+    public class Startup
+    {
+        public void Configure(IApplicationBuilder app)
+        {
+            app.UseStatusCodePages(new StatusCodePagesOptions().WithReExecute("/errors/{0}"));
```

Comment:
Proposed comment:

`// Register a handler for handling status code based error page. This example ends up using MVC to show an error page.`